hover/src/main/java/io/mattcarroll/hover/defaulthovermenu/window/WindowHoverMenu.java

```diff
@@ -45,6 +45,7 @@
     private boolean mIsShowingHoverMenu; // Are we currently display mHoverMenuView?
     private boolean mIsInDragMode; // If we're not in drag mode then we're in menu mode.
     private Set<OnExitListener> mOnExitListeners = new HashSet<>();
+    private Set<OnCollapseAndExpandListener> mOnCollapseAndExpandListeners = new HashSet<>();
```

Comment:
Can you change the implementation to a CopyOnWriteArraySet to avoid possible concurrency issues?